okta/config.go

```diff
@@ -134,6 +135,12 @@ func WithToken(token string) ConfigSetter {
 	}
 }
 
+func WithClientAssertion(clientAssertion string) ConfigSetter {
```

Comment:
Should add an entry in the README where all the other `With*` config helpers are explained.

Reply:
https://github.com/okta/okta-sdk-golang#configuration-setter-object